### PR TITLE
Annikas survey

### PR DESCRIPTION
**Pull request by annsig**
Netlify link
https://voluble-squirrel-5a409b.netlify.app/

### CODE REVIEW

File: src/App.jsx
Comment (AntonellaMorittu) [inline src/App.jsx:1]: please format your file 🪄

File: src/components/Testcomponent.jsx
Comment (AntonellaMorittu) [inline src/components/Testcomponent.jsx:1]: please format to two-space tab indentation 🪄

File: src/components/pexels-quark-studio-2507010.jpg
Comment (AntonellaMorittu) [inline src/components/pexels-quark-studio-2507010.jpg:1]: please move inside assets folder instead 👍

**Review by AntonellaMorittu — APPROVED** [review]
Hello Annika and congrats with your Survey project! The app satisfies the requirements and also performs field validation. Well done with responsiveness and accessibility, I left some comments in the code, you did a good job! 👏

**Review by jyy009 — COMMENTED** [review]
Hi! Your survey looks great. Your code was easy to follow and understand. The slider you used for the rating is really cool! Nice work!

**Pull request closed without merging**